docs/contributing/versioning.md

```diff
@@ -0,0 +1,121 @@
+# Versioning
+
+With an user interface (UI) library there are many ways that people can consume our code.
+
+We follow [semantic versioning](https://semver.org/) but a UI library often has subjective changes such as visual spacing changes.
+
+This document aims to give a basis to make decisions as a team.
```

Comment:
It might help to clarify if we add something like: "on how to version our releases".